README.md

```diff
@@ -143,6 +154,25 @@ spec:
   # ...
 ```
 
+### Configuration
+
+Most of Osiris configuration is done with Kubernetes annotations.
+
+The following table lists the configurable annotations for each Kubernetes resource and their default values.
+
+Kind | Annotation | Description | Default 
```

Comment:
Horizontal real estate is always at a premium. Can we drop the `Kind` column and split this up into two tables-- one for deployments (or deployments / statefulsets in the near future) and another for services?

This would create more space for the lengthier description text on each row.